### PR TITLE
OCPBUGS#22472: Amend misleading reference to basic DDoS protection

### PR DESCRIPTION
**Pull request by kowen-rh**
Version(s):
4.11+

Issue:
[OCPBUGS-22472](https://issues.redhat.com/browse/OCPBUGS-22472)

Link to docs preview:
[Route-specific annotations](https://70540--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/routes/route-configuration#nw-route-specific-annotations_route-configuration)

QE review:
- [x] QE has approved this change.

Additional information:

N/A

### CODE REVIEW

**Comment by kowen-rh** [comment]
@ShudiLi PTAL. Thank you!

**Comment by ShudiLi** [comment]
/lgtm
thanks

File: modules/nw-route-specific-annotations.adoc [hunk 1]
```diff
@@ -27,13 +27,13 @@ and "-". The default is the hashed internal key name for the route. |
 |`haproxy.router.openshift.io/pod-concurrent-connections`| Sets the maximum number of connections that are allowed to a backing pod from a router. +
 Note: If there are multiple pods, each can have this many connections.  If you have multiple routers, there is no coordination among them, each may connect this many times. If not set, or set to 0, there is no limit. |
 |`haproxy.router.openshift.io/rate-limit-connections`| Setting `'true'` or `'TRUE'` enables rate limiting functionality which is implemented through stick-tables on the specific backend per route. +
-Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks. |
```
Comment (mramendi) [inline modules/nw-route-specific-annotations.adoc:30]: Are you sure you want to keep the DoS acronym? The RH style guide at https://redhat-documentation.github.io/supplementary-style-guide/#denial-of-service-adj just says "denial-of-service attack". Not a critical issue, please make the call here.

**Review by mramendi — COMMENTED** [review]
@JoeAldinger will review this peer review as I am onboarding on the squad

File: modules/nw-route-specific-annotations.adoc (hunk 1 above)
Comment (kowen-rh) [reply modules/nw-route-specific-annotations.adoc:30]: Thanks for pointing that out! We also don't use it anywhere else that I'm aware of, so I'll go ahead and remove that. :slightly_smiling_face:

**Pull request merged**